index.js

```diff
@@ -1,9 +1,13 @@
 'use strict';
-exports.topic = {
-  name: "pt",
-  description: "Search and/or tail logs"
-};
+var extend      = require('util')._extend;
+var command     = require('./commands/pt');
+var description = "Search and/or tail logs";
+
+// `pt` topic kept for backwards-compatibility.
+exports.topics = [{ name: "papertrail", description: description },
+                  { name: "pt",         description: description }];
```

Comment:
Is it worth appending ` (legacy)` to the `pt` description? No strong preference, but I could see pointing folks to the new canonical one.